src/views/product-landing/helpers/transform-raw-content-to-prop.ts

```diff
@@ -0,0 +1,60 @@
+import { ProductData } from 'types/products'
+// raw content types
+import { ProductLandingContent, ProductLandingContentBlock } from '../schema'
+// component prop utilities & types
+import { makeHeadingSlugScope } from './'
+import { ProductLandingViewProps } from '../types'
+import { ProductLandingBlock } from '../components/product-landing-blocks/types'
+
+export async function transformRawContentToProp(
```

Comment:
TLDR; takes raw content, makes it into exactly the props needed to render the view.

Intent here is to pre-emptively tackle the "passing more data than necessary" to the client issue that I so far have failed to fix in `Product Tutorials Landing`. This function will also handle fetching tutorial and collection data (in a subsequent PR). My hope is that parts of this may be useful as reference when we do go to optimize the `Product Tutorials Landing` server stuff 👍

Reply:
Love it